docs/en/Running-Alluxio-on-a-Cluster.md

```diff
@@ -25,12 +25,21 @@ $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
 ```
 
 Update `alluxio.master.hostname` in `conf/alluxio-site.properties` to the hostname of the machine
-you plan to run Alluxio Master on. Add the IP addresses of all the worker nodes to the
-`conf/workers` file. Finally, sync all the information to worker nodes. You can use
+you plan to run Alluxio Master on. Add the IP address or hostname of all the worker nodes to the
+`conf/workers` file. You cannot use local file system as Alluxio's under storage system if there
+are more workers than one. Instead you need to set up a shared storage to which all Alluxio servers
+have access. The shared storage can be network file system (NFS), HDFS, S3, and so on. For example,
+you can refer to [Configuring Alluxio with S3](Configuring-Alluxio-with-S3.html) and follow its
+instructions to set up S3 as Alluxio's under storage.
+
+Finally, sync all the information to worker nodes. You can use
```

Comment:
nit: to worker nodes -> to the worker nodes

Reply:
Done